modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+[id="running-the-performance-creator-profile-wrapper_{context}"]
+= Running the Performance Profile wrapper script
+
+The performance profile wrapper script simplifies the running of the Performance Profile Creator (PPC) tool. It hides the complexities associated with running `podman` and specifying the mapping directories
+and it enables the creation of the performance profile.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
```

Comment:
The tool does not need access to the cluster, we may need to emphasize it

Reply:
The tool only needs access to the PAO image and a must-gather tarball, nothing else.